src/main/kotlin/club/sk1er/mods/levelhead/bedwars/ProxyClient.kt

```
@@ -317,10 +317,10 @@ object ProxyClient {
             }
         } catch (ex: IOException) {
             notifyNetworkIssue(ex)
-            identifierToUuid.values.associateWith { FetchResult.TemporaryError(ex.message) }
+            identifierToUuid.values.associateWith { FetchResult.TemporaryError(ex.message?.sanitizeForLogs()) }
         } catch (ex: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)